docs/src/main/sphinx/security/apache-ranger-access-control.md

```diff
@@ -0,0 +1,168 @@
+    <description>Boolean flag. true: use only user-to-groups mapping from Apache Ranger; false: use user-to-groups mappings from Apache Ranger and Trino. Default: false</description>
+  </property>
+
+  <property>
+    <name>ranger.plugin.trino.super.users</name>
+    <value></value>
+    <description>Comma separated list of user names. Superusers will be authorized for all accesses, without requiring explicit policy grants.</description>
+  </property>
+
+  <property>
+    <name>ranger.plugin.trino.super.groups</name>
+    <value></value>
+    <description>Comma separated list of group names. Users in supergroups will be authorized for all accesses, without requiring explicit policy grants</description>
+  </property>
+</configuration>
+```
+
+### ranger-trino-audit.xml
+```
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration xmlns:xi="http://www.w3.org/2001/XInclude">
+  <property>
+    <name>xasecure.audit.is.enabled</name>
+    <value>true</value>
+    <description>Boolean flag to specify if the plugin should generate access audit logs. Default: true</description>
+  </property>
+
+  <property>
+    <name>xasecure.audit.solr.is.enabled</name>
+    <value>false</value>
+    <description>Boolean flag to specify if audit logs should be stored in Solr. Default: false</description>
+  </property>
+
+  <property>
+    <name>xasecure.audit.solr.solr_url</name>
+    <value></value>
+    <description>URL to Solr deployment where the plugin should send access audits to</description>
+  </property>
+</configuration>
+```
+
+### ranger-policymgr-ssl.xml
+```
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration xmlns:xi="http://www.w3.org/2001/XInclude">
+  <!-- properties used for 2-way SSL between the Trino plugin and Apache Ranger server -->
+  <property>
+    <name>xasecure.policymgr.clientssl.keystore</name>
+    <value></value>
+    <description>Path to keystore file</description>
+  </property>
+
+  <property>
+    <name>xasecure.policymgr.clientssl.keystore.type</name>
+    <value>jks</value>
+    <description>Type of keystore. Default: jks</description>
+  </property>
+
+  <property>
+    <name>xasecure.policymgr.clientssl.keystore.credential.file</name>
+    <value></value>
+    <description>Path to credential file for the keystore; the credential should be in alias sslKeyStore</description>
+  </property>
+
+  <property>
+    <name>xasecure.policymgr.clientssl.truststore</name>
+    <value></value>
+    <description>Path to truststore file</description>
+  </property>
+
+  <property>
+    <name>xasecure.policymgr.clientssl.truststore.type</name>
+    <value>jks</value>
+    <description>Type of truststore. Default: jks</description>
+  </property>
+
+  <property>
+    <name>xasecure.policymgr.clientssl.truststore.credential.file</name>
+    <value></value>
+    <description>Path to credential file for the truststore; the credential should be in alias sslTrustStore</description>
```

Comment:
This file creates challenges with secrets from files in some environments this is inconsistent with:

```
http-server.https.truststore.path=/mnt/shared/certs/localcacerts
http-server.https.truststore.key=<truststore-password>
```
Detailed here https://trino.io/docs/current/security/inspect-jks.html

@kokosing @mosabua what do you think

Can this be updated to use secrets from Env variables which would therefore be consistent with 
https://trino.io/docs/current/security/secrets.html

Reply:
@lozbrown  - supporting this Trino way of handling secrets will require enhancements in Apache Ranger library. I suggest tracking this in Apache Ranger community, and update the plugin once a Apache Ranger release includes the enhancements. This shouldn't be blocker for this PR.

Reply:
I'm very eager to see merged soon too...I watched the contributor call this morning hoping this would have come up but alas no. I agree that at this point I'd rather see it merged as is.

Getting that secret file into the pod without exposing the secret in source control,  will be a pain in the proverbial but we'll work it out.

Is there any way not to need that if we're only using 1 way SSL and the ca certs in trust store are not secrets?